src/app/static/src/app/components/Stepper.vue

```diff
@@ -154,7 +157,21 @@
                     return true
                 }
                 return !this.isComplete(activeStep)
-            }
+            },
+            clearWarnings(){
+                const mutationMethods: { [key: number]: () => void; } = {
+                    3: this.clearModelOptionsWarnings,
+                    4: this.clearModelRunWarnings,
+                    5: this.clearModelCalibrateWarnings,
+                    6: this.clearModelCalibrateWarnings,
```

Comment:
```suggestion
                    6: this.clearModelCalibrateWarnings,
                    7: this.clearModelCalibrateWarnings
```

Reply:
Sorry, should have spotted this little bug before - calibrate warnings couldn't be dismissed from the final step, think you just need to add this line.